test/integration/suite/traefik_test.go

```diff
@@ -116,8 +118,18 @@ func TestTraefikIngressRouteAndMiddleware(t *testing.T) {
 		t.Logf("Error building kubeconfig: %v", err)
 	}
 
+	// Create a new scheme
+	schemeBuilder := &scheme.Builder{GroupVersion: traefikv1alpha1.SchemeGroupVersion}
+	schemeBuilder.Register(&traefikv1alpha1.Middleware{}, &traefikv1alpha1.MiddlewareList{})
+	schemeBuilder.Register(&traefikv1alpha1.IngressRoute{}, &traefikv1alpha1.IngressRouteList{})
+
+	myScheme, err := schemeBuilder.Build()
```

Comment:
nice name :D would you keep or make it more nginx related?

Reply:
ourScheme? 😆